docs/skopeo-list-tags.1.md

```diff
@@ -72,6 +72,9 @@ This commands refers to repositories using a _transport_`:`_details_ format. The
         "docker.io/myuser/myimage:v1.0"
```

Comment:
Above, the text “Currently, only the Docker transport is supported.” needs to be updated (probably just deleted).

Reply:
this had done